### PR TITLE
Fix | Members | See Information

### PR DESCRIPTION
**Pull request by Yopo-Nicolas**

### CODE REVIEW

File: src/Views/AgencyGroups/ViewGroupModal/GroupInfoMembers/MemberList/MemberList.tsx [hunk 1]
```diff
@@ -9,16 +9,22 @@ import { SFCollapse } from 'sfui';
 export interface MemberListProps {
   isActive: boolean;
   members: GroupMember[];
+  onClick?: (id: string) => void;
```
Comment (martinweingart) [inline src/Views/AgencyGroups/ViewGroupModal/GroupInfoMembers/MemberList/MemberList.tsx:12]: Should this be optional?

File: src/Views/AgencyShifts/ShiftInfoModal/ShiftInfoModal.tsx [hunk 2]
```diff
@@ -97,6 +106,20 @@ export const ShiftInfoModal = ({
     }
   };
 
+  const onSeeMemberInformation = async (item: ShiftMember) => {
+    setIsSaving(true);
```
Comment (martinweingart) [inline src/Views/AgencyShifts/ShiftInfoModal/ShiftInfoModal.tsx:110]: Change `isSaving` to a more generic name or use another state

**Review by martinweingart — CHANGES_REQUESTED** [review]

File: src/Views/AgencyShifts/ShiftInfoModal/ShiftInfoModal.tsx (hunk 2 above)
Comment (Yopo-Nicolas) [reply src/Views/AgencyShifts/ShiftInfoModal/ShiftInfoModal.tsx]: Changed to isModalLoading

File: src/Views/AgencyGroups/ViewGroupModal/GroupInfoMembers/MemberList/MemberList.tsx (hunk 1 above)
Comment (Yopo-Nicolas) [reply src/Views/AgencyGroups/ViewGroupModal/GroupInfoMembers/MemberList/MemberList.tsx]: Removed optional

**Review by martinweingart — APPROVED** [review]
LGTM

**Pull request merged**